### PR TITLE
Update dynflow to 0.7.8

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

**Comment by domcleal** [comment]
@iNecas please also add a changelog entry

**Comment by domcleal** [comment]
And update gem.list.

**Comment by iNecas** [comment]
Updated

File: plugins/ruby-foreman-dynflow/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+ruby-foreman-dynflow (0.7.8-1) stable; urgency=low
+
+  * Update to 0.7.8
+
 ruby-foreman-dynflow (0.7.7-1) stable; urgency=low
```
Comment (domcleal) [inline plugins/ruby-foreman-dynflow/debian/changelog]: Missing a line with your name + date here.  `scripts/changelog.rb` may help you.

Comment (iNecas) [inline plugins/ruby-foreman-dynflow/debian/changelog]: It helped indeed :)

**Comment by domcleal** [comment]
Merged as 3e80c87 for nightlies, thanks @iNecas!

**Pull request closed without merging**